src/ray/gcs/gcs_client/gcs_client.h

```diff
@@ -53,7 +53,10 @@ class GcsClientOptions {
         password_(password),
         enable_sync_conn_(enable_sync_conn),
         enable_async_conn_(enable_async_conn),
-        enable_subscribe_conn_(enable_subscribe_conn) {}
+        enable_subscribe_conn_(enable_subscribe_conn) {
+    RAY_LOG(DEBUG) << "Connect to gcs server via redis: " << redis_ip << ":"
+                   << redis_port;
+  }
```

Comment:
Should be removed?

Reply:
It's debugging log. It's useful for debugging purposes. So I'd prefer leaving here.